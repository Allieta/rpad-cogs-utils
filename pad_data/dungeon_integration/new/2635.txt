Cloud City's Scarlet Blaze - 4th Floor-Leader Skills Invalid - 1

stage 0

689 - Rowdy Red Samurai Goblin @ level 7
----- When 1 enemy remains -----
Offensive Formation
Increase damage to 200% for the next 99 turns
----- HP <= 100 -----
Wild Slashes
13834 Damage - Deal 90%~180% damage (3~6 hits, 30% each) (70% chance)

690 - Marine Spec Ops Goblin @ level 7
Charge
5856 Damage - Deal 100% damage
----- HP <= 100 -----
Wild Strikes
11712 Damage - Deal 50%~200% damage (2~8 hits, 25% each)

691 - Heavy Infantry Hobgoblin @ level 7
----- HP <= 100 -----
Big Blade
16622 Damage - Deal 100% damage

stage 1

475 - Hellfire Pyro Demon @ level 5
----- HP <= 100 -----
Water Bind
Bind Water cards for 2~4 turns (30% chance)
----- HP <= 35 -----
Evil Slash
18337 Damage - Deal 180% damage (6 hits, 30% each) (40% chance)

476 - Hellice Frost Demon @ level 5
----- HP <= 100 -----
Leaf Bind
Bind Wood cards for 2~4 turns (30% chance)
----- HP <= 35 -----
Evil Slash
18075 Damage - Deal 180% damage (6 hits, 30% each) (40% chance)

477 - Hellwind Taur Demon @ level 5
----- HP <= 100 -----
Fire Bind
Bind Fire cards for 2~4 turns (30% chance)
----- HP <= 35 -----
Evil Slash
17775 Damage - Deal 180% damage (6 hits, 30% each) (40% chance)

478 - Hellray Harpie Demon @ level 5
----- HP <= 100 -----
Dark Bind
Bind Dark cards for 2~4 turns (30% chance)
----- HP <= 35 -----
Evil Slash
18637 Damage - Deal 180% damage (6 hits, 30% each) (40% chance)

479 - Helldark Blood Demon @ level 5
----- HP <= 100 -----
Flash Bind
Bind Light cards for 2~4 turns (30% chance)
----- HP <= 35 -----
Evil Slash
18900 Damage - Deal 180% damage (6 hits, 30% each) (40% chance)

stage 2

263 - Magma Chimera @ level 7
----- HP <= 75 -----
Double Strike
20752 Damage - Deal 140% damage (2 hits, 70% each) (40% chance)
Double Bind
Bind 2 random cards for 2~4 turns (40% chance) (1 time use)

265 - Deep Chimera @ level 7
----- HP <= 75 -----
Double Strike
20453 Damage - Deal 140% damage (2 hits, 70% each) (40% chance)
Double Bind
Bind 2 random cards for 2~4 turns (40% chance) (1 time use)

267 - Flower Chimera @ level 7
----- HP <= 75 -----
Double Strike
20111 Damage - Deal 140% damage (2 hits, 70% each) (40% chance)
Double Bind
Bind 2 random cards for 2~4 turns (40% chance) (1 time use)

269 - Thunder Chimera @ level 7
----- HP <= 75 -----
Double Strike
21093 Damage - Deal 140% damage (2 hits, 70% each) (40% chance)
Double Bind
Bind 2 random cards for 2~4 turns (40% chance) (1 time use)

271 - Fog Chimera @ level 7
----- HP <= 75 -----
Double Strike
21392 Damage - Deal 140% damage (2 hits, 70% each) (40% chance)
Double Bind
Bind 2 random cards for 2~4 turns (40% chance) (1 time use)

stage 3

221 - Berserk @ level 3
----- Turn 1 -----
Hyper Fire Bind
Bind Fire cards for 10 turns (1 time use)
----- HP <= 25 -----
Clean Cut
42254 Damage - Deal 200% damage

223 - Highlander @ level 3
----- Turn 1 -----
Dark Flash
Blind all orbs on the board (1 time use)
----- HP <= 75 -----
Legion Thrust
20555 Damage - Deal 120%~240% damage (3~6 hits, 40% each) (70% chance)

