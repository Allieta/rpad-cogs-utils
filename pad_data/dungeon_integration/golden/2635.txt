Cloud City's Scarlet Blaze - 4th Floor-Leader Skills Invalid

stage 0

690 - Marine Spec Ops Goblin @ level 7

Charge
Deal 100% damage
----- HP <= 100 -----
HP <= 100
Wild Strikes
Deal 50%~200% damage (2~8 hits, 25% each)

691 - Heavy Infantry Hobgoblin @ level 7

----- HP <= 100 -----
HP <= 100
Big Blade
Deal 100% damage

689 - Rowdy Red Samurai Goblin @ level 7

----- Turn 2 -----
Turn 2
Offensive Formation
Increase damage to 200% for the next 99 turns
----- HP <= 100 -----
HP <= 100
Wild Slashes
Deal 90%~180% damage (3~6 hits, 30% each)

stage 1

475 - Hellfire Pyro Demon @ level 5

----- HP <= 100 -----
HP <= 100
Water Bind
Bind Water cards for 2~4 turns
----- HP <= 34 -----
HP <= 34
Water Bind
Bind Water cards for 2~4 turns
----- HP <= 35 -----
HP <= 35
Evil Slash
Deal 180% damage (6 hits, 30% each)

476 - Hellice Frost Demon @ level 5

----- HP <= 100 -----
HP <= 100
Leaf Bind
Bind Wood cards for 2~4 turns
----- HP <= 34 -----
HP <= 34
Leaf Bind
Bind Wood cards for 2~4 turns
----- HP <= 35 -----
HP <= 35
Evil Slash
Deal 180% damage (6 hits, 30% each)

478 - Hellray Harpie Demon @ level 5

----- HP <= 100 -----
HP <= 100
Dark Bind
Bind Dark cards for 2~4 turns
----- HP <= 34 -----
HP <= 34
Dark Bind
Bind Dark cards for 2~4 turns
----- HP <= 35 -----
HP <= 35
Evil Slash
Deal 180% damage (6 hits, 30% each)

477 - Hellwind Taur Demon @ level 5

----- HP <= 100 -----
HP <= 100
Fire Bind
Bind Fire cards for 2~4 turns
----- HP <= 34 -----
HP <= 34
Fire Bind
Bind Fire cards for 2~4 turns
----- HP <= 35 -----
HP <= 35
Evil Slash
Deal 180% damage (6 hits, 30% each)

479 - Helldark Blood Demon @ level 5

----- HP <= 100 -----
HP <= 100
Flash Bind
Bind Light cards for 2~4 turns
----- HP <= 34 -----
HP <= 34
Flash Bind
Bind Light cards for 2~4 turns
----- HP <= 35 -----
HP <= 35
Evil Slash
Deal 180% damage (6 hits, 30% each)

stage 2

267 - Flower Chimera @ level 7

This monster will only use basic attacks (or computing enemy skills failed)
271 - Fog Chimera @ level 7

This monster will only use basic attacks (or computing enemy skills failed)
269 - Thunder Chimera @ level 7

This monster will only use basic attacks (or computing enemy skills failed)
263 - Magma Chimera @ level 7

This monster will only use basic attacks (or computing enemy skills failed)
265 - Deep Chimera @ level 7

This monster will only use basic attacks (or computing enemy skills failed)
stage 3

223 - Highlander @ level 3

----- Turn 1 -----
Turn 1
Dark Flash
Blind all orbs on the board
----- HP <= 75 -----
HP <= 75
Legion Thrust
Deal 120%~240% damage (3~6 hits, 40% each)

221 - Berserk @ level 3

----- Turn 1 -----
Turn 1
Hyper Fire Bind
Bind Fire cards for 10 turns
----- HP <= 25 -----
HP <= 25
Clean Cut
Deal 200% damage

